cypress/e2e/Hw14Tests/hw14Tests.cy.js

```diff
@@ -0,0 +1,25 @@
+describe("Hw 014 test 1", () => {
```

Comment:
1. create meaningful titles for describes and tests
2. these tests should be places in different files